neurons/nodes/bitcoin/node.py

```diff
@@ -111,3 +118,122 @@ def get_address_and_amount_by_txn_id_and_vout_id(self, txn_id: str, vout_id: str
         else: # get from hash table if exists
             address, amount = self.tx_out_hash_table[txn_id[:3]][(txn_id, vout_id)]
             return address, int(amount)
+
+    def create_challenge(self, start_block_height, last_block_height):
+        block_to_check = random.randint(start_block_height, last_block_height)
```

Comment:
this can return duplicates, maybe sample instead?

Reply:
This will return a integer value between the 2 range inclusive. there is no risk for duplicate